doc/protocols/sigprefixlist/SigPrefixListProtocol.md

```diff
@@ -0,0 +1,71 @@
+# SIG Prefix List Protocol
+
+## Overview
+The protocol is designed to enable SIGs to automatically fetch IP prefix to AS ID mappings. It uses a distributed coordinated infrastructure with the following design and security goals. 
+
+The system should
+- Be reselient to hijacking attacks 
+- Be resilient to flooding attacks on SIGs
+- Be resilient to downgrade attacks to IP internet where SCION connections are possible
+- Not be a single point of failure
```

Comment:
>Not be a single point of failure

Not be a single point of failure for what? The SIG operation in general?
Or should the system itself not have single points of failure in the services it requires and provides?

Reply:
What I meant was that the system should not have components that could be single points of failure. I added more description to the file